app/controllers/application_controller.rb

```diff
@@ -69,6 +69,10 @@ def prompt_to_enter_otp
     redirect_to user_two_factor_authentication_url
   end
 
+  def create_event(event_type, user = current_user)
+    Event.create_from_request(user, event_type, request)
+  end
+
```

Comment:
We might want to differentiate this method further since we have the [`Analytics`](https://github.com/18F/identity-idp/blob/bc85a44b5a2d829794938704c6cb7943ef442fe2/app/services/analytics.rb) class and associated events. These events are specific to logged in users.